char_index.py

```diff
@@ -0,0 +1,29 @@
+char_position = 0
+lowecase_chars = ['countat1','a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+return_message = "Position of alphabet: {char_position}"
```

Comment:
do we really want to return a whole message instead of just the value and let the caller use it how it likes?

Reply:
Requirement from the codeacademy was to return the full string as above.

Reply:
ah okay rgr that.